src/main/scripts/ci/before-install.sh

```diff
@@ -4,6 +4,9 @@
 echo       Running $0
 echo *-*-*-*-*-*-*-*-*-*-*-*-*-*
 
+# loging to docker image repository
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
we have no docker image inside, what is it?

Reply:
nothing